dev/io.openliberty.http.monitor_fat/test-applications/JsfApp/resources/index.xhtml

```diff
@@ -0,0 +1,17 @@
+<?xml version='1.0' encoding='UTF-8' ?>
```

Comment:
missing copyright

Reply:
I don't think we usually copyright HTML files?

Reply:
Example: https://github.ibm.com/websphere/WS-CD-Open/blob/094176f81aea6ff54531d017f672d6fd13e4988b/dev/com.ibm.ws.jsf_fat_lWAS/test-applications/JSF20Tags.war/resources/index.html